video/out/hwdec/dmabuf_interop_gl.c

```diff
@@ -160,15 +160,15 @@ static void vaapi_gl_mapper_uninit(const struct ra_hwdec_mapper *mapper)
 static bool vaapi_gl_map(struct ra_hwdec_mapper *mapper, bool probing)
 {
     struct priv *p_mapper = mapper->priv;
-    struct priv_owner *p_owner = mapper->owner->priv;
+    struct dmabuf_interop *p_owner = mapper->owner->priv;
```

Comment:
Isn't `mapper->owner` still `struct priv_owner` in `hwdec_vaapi.c`, of which `struct dmabuf_interop` is *not* the first member?